packages/cli/src/commands/generate-graphql.ts

```diff
@@ -56,19 +58,19 @@ export default class GenerateGraphql extends Command {
     })
 
     runCommandSync({
-      cmd: 'yarn format:generated',
+      cmd: `${packageManager} run format:generated`,
       errorMessage:
-        "Failed to format generated files. 'yarn format:generated' thrown errors",
+        `Failed to format generated files. '${packageManager} format:generated' thrown errors`,
       throws: 'warning',
       debug,
       cwd: isCore ? undefined : tmpDir,
     })
 
-    // yarn generate:copy-back expects the DESTINATION var to be present so it can copy the files to the correct directory
+    // ${packageManager} generate:copy-back expects the DESTINATION var to be present so it can copy the files to the correct directory
```

Comment:
suggestion: for clarity on the comment, what do you think?
```suggestion
    // The command generate:copy-back expects the DESTINATION var to be present so it can copy the files to the correct directory
```

Reply:
@lariciamota I decided to leave `${packageManager}` on the comment thinking that leaving the handle alone could cause some confusion, but your suggestion makes sense. I'll update accordingly. Thanks!